src/plugins/displayLayout/components/LayoutItem.vue

```diff
@@ -208,24 +138,30 @@
                 this.updatePosition(event);
 
                 if (this.activeDrag) {
-                    this.$emit('dragInProgress', this.item.id, this.activeDrag.getAdjustedPosition(this.delta));
+                    this.$emit(
+                        'dragInProgress',
+                        this.item,
+                        this.activeDrag.getAdjustedPosition(this.delta)
+                    );
                 }
             },
             endDrag(event) {
                 document.body.removeEventListener('mousemove', this.continueDrag);
                 document.body.removeEventListener('mouseup', this.endDrag);
                 this.continueDrag(event);
-                this.$emit('endDrag', this.item.id);
+                this.$emit('endDrag', this.item);
                 this.initialPosition = undefined;
                 event.preventDefault();
             }
         },
         mounted() {
+            let context = {};
```

Comment:
why not
```
let context = {
    item: this.item.domainObject,
    layoutItem = this.item
}
``` 
?

Reply:
At some point I tried to build the context conditionally before deciding to pass the layoutItem instead, and it's a left over code. Thanks for pointing it out. It's fixed.